erizo/src/erizo/logger.h

```diff
@@ -101,4 +102,15 @@
 #define ELOG_FATAL(fmt, args...) \
     ELOG_FATAL2( logger, fmt, ##args );
 
+static const std::string LOG_CONTEXT_ID = "connId";
```

Comment:
could it be called just "id"? or "componentId"? I'm saying that because "connId" seems to refer to connection and I'm not sure it will be always the case.